### DISCUSSION TITLE
Create Basic Presets

### DISCUSSION THREAD

**Discussion by cpreston321** [danielroe/magic-regexp#48] [Ideas]
# 💡 Create Presets
 
Create basic regex presets for the most commonly used regex like 

- US Phone Numbers 
- Email Validation 
- etc 

Might seem like a simple regex but would be nice just to have something that exist and ready to go 😄 

Great work on this package! 

Thanks!! ⭐

**Comment by aarondill** [comment]
To be clear, email validation *correctly* is no simple Regex. [Here](https://www.netmeister.org/blog/email.html) is a great (slightly long) article which goes through the actual specs and explains why even the most complex regexes don’t generally match the spec. I’m not saying it couldn’t be helpful, just that extra thought should be taken if including this in a library.
